README.md

```diff
@@ -16,9 +16,6 @@
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue)](https://github.com/feast-dev/feast/blob/master/LICENSE)
 [![GitHub Release](https://img.shields.io/github/v/release/feast-dev/feast.svg?style=flat&sort=semver&color=blue)](https://github.com/feast-dev/feast/releases)
 
-## Join us on Slack!
```

Comment:
Can you revert this change?

Reply:
Ah sorry - don't know where that came from - reverted it!

Reply:
Okay I was able to remove this but only using `--no-verify`